thesis/03-implementation.tex

```diff
@@ -1,44 +1,43 @@
 \subsection{Prototype}
 
-From the list of features mentioned in the previous chapter we have selected the most important ones and created a prototype in a form of MVP (Minimum Viable Product). 
+From the list of requirements mentioned in the previous chapter we have selected the most important ones and created a prototype that would serve as a MVP (Minimum Viable Product) which we could develop further into a full-featured application.
+\smallskip
 
-The selected features:
+The selected features list consists of: 
 
 \begin{enumerate}
-	\requirement{3}{load an Instruction}
-	\requirement{3}{view the 3D representation of the Instruction step}
-	\requirement{3}{switch to the next step in the Instruction}
-	\requirement{3}{switch to the previous step in the Instruction}
-	\requirement{3}{see the Transition between two Instruction steps}
-	\requirement{2}{pause the Transition at any time}
-	\requirement{2}{rewind the Transition}
-	\requirement{2}{forward the Transition}
-	\requirement{3}{rotate the Model in 3D space}
-	\requirement{3}{zoom the Model in and out in 3D space}
-	\requirement{1}{see creases of the Model} 
-	\requirement{2}{distinguish paper's top and bottom sides} 
+	\requirement{3}{loading an Instruction}
```

Comment:
i believe we are missing commas at the end of each item.

Reply:
`\requirement` macro adds dot at the end of each one